cajita/src/Cajita_SparseArray.hpp

```diff
@@ -475,6 +478,23 @@ class SparseArray
         return _data.access( tile_id );
     }
 
+    // ------------------------------------------------------------------------
+    // data access
+    // aceess tuple
```

Comment:
It's just a tag showing this part is for tuple access, and the next part is for element access. I feel it could be a tag to organize interfaces for later extensions, if there are any.